source/modules/MEI.analytical.xml

```diff
@@ -329,6 +329,7 @@
   <classSpec ident="att.nc.anl" module="MEI.analytical" type="atts">
     <desc>Analytical domain attributes.</desc>
     <classes>
```

Comment:
An accidental is not part of the analytical domain.

Reply:
Hi @ahankinson. I was looking into the documentation for the `<note>` element (the equivalent to the `<nc>` element in CMN). It seems that the attribute `@accid` is coming from the [analytical domain (`att.note.anl`)](https://music-encoding.org/guidelines/dev/attribute-classes/att.note.anl.html). Is this correct? 

I recommended adding this attribute as part of the analytical domain because that seemed to be the case for notes (`att.note.anl` gets it from `att.accidental`; here `att.nc.anl` gets it from that same class).

Reply:
OK; that makes a certain amount of sense for consistency. Still, it might make sense to use `att.accidental` directly on the element that requires it, instead of going through the `.anl` class. Let's see what others say.